app/mailers/sponsor_group_mailer.rb

```diff
@@ -0,0 +1,17 @@
+class SponsorGroupMailer < ActionMailer::Base
+  default :from => "noreply@uOttawa.ca"
+  ADMIN_EMAIL = "RefugeeSSP@uOttawa.ca"
+
+  def intake_form_submitted(sponsor_group)
+    @sponsor_group = sponsor_group
+
+    @app_url = case Rails.env
+    when 'production'
```

Comment:
Indent `when` as deep as `case`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.